cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -380,6 +490,132 @@ writeLedgerState mOutFile qState@(SerialisedLedgerState serLedgerState) =
      first (const ls) (decodeFull ls)
 
 
+-- Data Structure to hold stake information so that it can be printed nicely
+data Stakes =  Stakes {
+      mark, set, go, total :: Integer
+    } deriving Show
+
+instance ToJSON Stakes where
+    -- this generates a Value
```

Comment:
`-- this generates a Value` is unnecessary

Reply:
OK.  I copied the comment from another instance